osgp/protocol-adapter-iec60870/osgp-protocol-adapter-iec60870/src/main/java/org/opensmartgridplatform/adapter/protocol/iec60870/infra/messaging/processors/GetHealthStatusRequestMessageProcessor.java

```diff
@@ -60,8 +60,9 @@ public void process(final ClientConnection deviceConnection, final RequestMetada
             this.getLoggingService().log(logItem);
 
         } catch (final IOException | RuntimeException e) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1166)